doc/help/pypeit_install_ql_masters.rst

```diff
@@ -14,6 +14,10 @@
                          directory. (default: None)
       --odir ODIR        The directory in which to extract the zip file. Ignored if
                          a direct path is provided using --ql_path. (default:
+<<<<<<<<< Temporary merge branch 1
```

Comment:
@suksien -- Looks like you had a bad merge for these `rst` files.
 Please fix them.

Reply:
Done